requirements/quality.txt

```diff
@@ -1,9 +1,9 @@
 # Requirements for code quality checks
 
-isort==4.3.4
-astroid==1.5.2
-pycodestyle==2.4.0
-pydocstyle==2.1.1
-pylint==1.7.1
-caniusepython3==7.0.0
-edx_lint==0.5.5
+isort==4.3.21
+astroid==2.3.3
```

Comment:
@taniwha, you can update the CircleCI configuration to use the Python 3.5 image instead of Python 2.7.